_posts/2021-02-19-cpp-version.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: USBGuard 
```

Comment:
I would change the title for something else, e.g.: "Increasing compiler version to C++17"